build/build.sh

```diff
@@ -0,0 +1,23 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+
+REMOTE_URL=https://github.com/secure-device-onboard/supply-chain-tools.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ];
```

Comment:
The indent level of 'then' should be at the same level as 'if' or in the same line. Indent should be 2-spaces.

Reply:
Changed indentation to 2-spaces format.